sdk-api-src/content/bcrypt/nf-bcrypt-bcryptduplicatekey.md

```diff
@@ -63,16 +63,20 @@ The handle of the key to duplicate. This must be a handle to a symmetric key.
 
 A pointer to a <b>BCRYPT_KEY_HANDLE</b> variable that receives the handle of the duplicate key. This handle is used in subsequent functions that require a key, such as <a href="/windows/desktop/api/bcrypt/nf-bcrypt-bcryptencrypt">BCryptEncrypt</a>. This handle must be released when it is no longer needed by passing it to the <a href="/windows/desktop/api/bcrypt/nf-bcrypt-bcryptdestroykey">BCryptDestroyKey</a> function.
 
-### -param pbKeyObject [out]
+### -param pbKeyObject [out, optional]
```

Comment:
The headers with param info are generated by our build system based on what's in the SDK. Changing it here won't impact what gets published to the Learn site.

Reply:
Hi @samuel-lee-msft. The rest of this PR looks good, but we can't change the signature of this parameter here. These are generated by the docs build system and any changes here will either be ignored or cause a build error. I can merge the rest of the changes if you can back out this part.

Reply:
I see the param type in bcrypt.h is defined as `_Out_writes_bytes_all_opt_(cbKeyObject)  PUCHAR   pbKeyObject`. I don't know that our build system has the ability to indicate that properly when creating the documentation header sections. I can discuss it with our engineering team at our next status meeting. For now, I would suggest adding some text to the description of the parameter below to indicate that it's optional.

Reply:
```suggestion
### -param pbKeyObject [out]
```

Reply:
W.r.t. the doc build system - I personally think it would be great to directly bring the SAL annotations to the learn site, rather than translating to [in] / [out]

Reply:
I agree. I'll open a work item with that team and discuss it at our next sync with them. Thanks for bringing it to my attention.